### DISCUSSION TITLE
Add SLANGP shader support.

### DISCUSSION THREAD

**Discussion by TheSystemGuy1337** [86Box/86Box#4970] [Ideas]
In short, recently an [impressive](https://github.com/kokoko3k/koko-aio-slang) CRT shader was released which 86box could benefit from but is SLANGP only. If it is even remotely possible, we should add support for those shaders. We could achieve this by using librashader which has OpenGL 3 support.

**Comment by mr-falafel** [comment]
Mentioned in my own post, here: https://github.com/86Box/86Box/discussions/5255#discussion-8012521
Libretro has implementations for slang shaders in OpenGL and Vulkan, which could maybe be used as reference.
